src/fusion/road_fuser.py

```diff
@@ -0,0 +1,111 @@
+"""BGT Data Fuser"""
+
+import numpy as np
+import ast
+import logging
+from shapely.geometry import Polygon
+
+from ..utils import clip_utils
+from ..utils.las_utils import get_bbox_from_tile_code
+from ..labels import Labels
+from ..fusion.bgt_fuser import BGTFuser
+
+logger = logging.getLogger(__name__)
+
+
+class BGTRoadFuser(BGTFuser):
+    """
+    Data Fuser class for automatic labelling of road points using BGT data
+    in the form of polygons. Data files are assumed to be in CSV format and
+    contain six columns: [bgt_name, polygon, x_min, y_max, x_max, y_min].
+
+    Parameters
+    ----------
+    label : int
+        Class label to use for this fuser.
+    bgt_file : str or Path or None (default: None)
+        File containing data files needed for this fuser. Either a file or a
+        folder should be provided, but not both.
+    bgt_folder : str or Path or None (default: None)
+        Folder containing data files needed for this fuser. Data files are
+        assumed to be prefixed by "bgt_buildings", unless otherwise specified.
+        Either a file or a folder should be provided, but not both.
+    file_prefix : str (default: 'bgt_building')
+        Prefix used to load the correct files; only used with bgt_folder.
+    road_offset : int (default: 0)
+        The road polygon will be extended by this amount (in meters).
+    padding : float (default: 0)
+        Optional padding (in m) around the tile when searching for objects.
+    """
+
+    COLUMNS = ['bgt_name', 'Polygon', 'x_min', 'y_max', 'x_max', 'y_min']
+
+    def __init__(self, label, bgt_file=None, bgt_folder=None,
```

Comment:
Misschien mooi om een (set) target_fields ofzo mee te geven met de specifieke road parts die we willen labellen? Dan kunnen we deze fuser later makkelijk gebruiken voor andere road parts, e.g. fietspaden etc.